### PR TITLE
"Adding code for nc_news_6"

### PR DESCRIPTION
**Pull request by zamarz**

### CODE REVIEW

File: __tests__/index.tests.js [hunk 1]
```diff
@@ -113,3 +113,49 @@ describe("GET /api/articles/:article_id", () => {
...
+      });
+  });
+
+  test("Get:200 ensure that the most recent comments appear first in the result", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at");
```
Comment (B7Bprog) [inline __tests__/index.tests.js]: Please check for descending order here.

File: __tests__/index.tests.js [hunk 2]
```diff
@@ -113,3 +113,49 @@ describe("GET /api/articles/:article_id", () => {
...
+      });
+  });
```
Comment (B7Bprog) [inline __tests__/index.tests.js:188]: Please write another test with status 200, where the article exists, but there are no comments. Expect an empty array.

File: db/controllers/news-controllers.js [hunk 3]
```diff
@@ -27,4 +31,15 @@ const getArticle = (request, response, next) => {
     });
 };
 
-module.exports = { getTopics, getEndpoints, getArticle };
+const getComments = (request, response, next) => {
+  const { article_id } = request.params;
```
Comment (B7Bprog) [inline db/controllers/news-controllers.js:48]: You will need to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.
You will need to check whether the article exists in order to determine this.
I recommend the use of 'Promise.all()' here.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Zach, please see my comments and refactor your code, then send us another PR with it.

File: db/models/news-models [hunk 4]
```diff
@@ -32,7 +32,7 @@ const findArticle = (article_id) => {
     });
 };
 
-const findComments = (article_id, sort_by = "created_at", order = "asc") => {
+const findComments = (article_id, sort_by = "created_at", order = "desc") => {
```
Comment (Augs0) [inline db/models/news-models:35]: This file doesn't appear to be a JS file. It's missing the .js extension

Comment (Augs0) [reply db/models/news-models:35]: Did you run your tests before committing? Because they likely would have failed due to this issue 😅

**Review by Augs0 — CHANGES_REQUESTED** [review]
Well done for following the feedback :) Just be careful with the model file which is not currently a js file...please fix this the you are okay to merge

File: db/models/news-models (hunk 4 above)
Comment (zamarz) [reply db/models/news-models:35]: I completely missed that, woops! I promise that all my tests were working, I've changed it now and they still seem to work thankfully :sweat_smile:

**Pull request merged**